keyring/keychain.go

```diff
@@ -2,27 +2,15 @@
 
 package keyring
 
-// https://developer.apple.com/library/mac/documentation/Security/Reference/keychainservices/index.html
-
-/*
-#cgo CFLAGS: -mmacosx-version-min=10.6 -D__MAC_OS_X_VERSION_MAX_ALLOWED=1060
-#cgo LDFLAGS: -framework CoreFoundation -framework Security
-
-#include <stdlib.h>
-#include <CoreFoundation/CoreFoundation.h>
-#include <Security/Security.h>
-*/
-import "C"
-
 import (
-	"errors"
 	"fmt"
 	"log"
-	"path/filepath"
-	"unicode/utf8"
-	"unsafe"
 
-	"github.com/mitchellh/go-homedir"
+	gokeychain "github.com/keybase/go-keychain"
+)
+
+const (
+	keychainAccessGroup = "ACE1234DEF.com.99designs.aws-vault"
```

Comment:
I don't fully understand what this is, but looks like it maps to [`kSecAttrAccessGroup`](https://developer.apple.com/documentation/security/ksecattraccessgroup) which doesn't sound necessary in this use-case, and `ACE1234DEF` looks suspiciously like sample data.

> Access groups are used to share keychain items among two or more apps.

It's being passed to `gokeychain.NewGenericPassword()` which calls [`SetAccessGroup()`](https://github.com/keybase/go-keychain/blob/69c518e5bfff89ad62db81a185634d3aeac3c453/keychain.go#L226) which calls [`k.SetString(AccessGroupKey, ag)`](https://github.com/keybase/go-keychain/blob/69c518e5bfff89ad62db81a185634d3aeac3c453/keychain.go#L188) which treats empty string as unset. So perhaps we should just pass an empty string to `gokeychain.NewGenericPassword()`? Or is this actually doing something?